tests/integration/customizations/s3/test_s3handler.py

```diff
@@ -375,5 +389,45 @@ def test_bucket(self):
         self.assertNotIn(self.bucket, buckets_list)
 
 
+class S3HandlerTestBucket(unittest.TestCase):
```

Comment:
Do we need this test since we have this one: https://github.com/aws/aws-cli/blob/develop/tests/integration/customizations/s3/test_plugin.py#L1083? I would prefer keeping tests out of this test suite because it is hard to work with and does not completely test end to end like ``test_plugin.py``